tests/jsmarionette/runner/marionette-js-runner/lib/ui.js

```diff
@@ -160,15 +160,20 @@ module.exports = function(suite) {
      */
     context.test = function(title, options, fn) {
       // Second argument is optional, put default device if it doesn't exist.
-      if (typeof(options) === 'function') {
-        fn = options;
+      if (typeof options !== 'object') {
+        if (typeof options === 'function') {
+          fn = options;
+        }
+
+        // Run only phone tests by default.
         options = { devices: ['phone'] };
```

Comment:
This makes it so that even if we just get one argument we still set `options` to the default